projects/noloan/app/src/main/java/hasadna/noloan/SmsRecyclerAdapter.java

```diff
@@ -53,28 +57,56 @@ public int getItemCount() {
   public class RecyclerViewHolder extends RecyclerView.ViewHolder {
 
     TextView from, content, receivedAt;
-    Button buttonSmsToLawsuit;
+    Button buttonCreateSmsLawsuit;
 
     public RecyclerViewHolder(@NonNull View itemView) {
       super(itemView);
       from = itemView.findViewById(R.id.received_from);
       content = itemView.findViewById(R.id.content);
       receivedAt = itemView.findViewById(R.id.receivedAt);
-      buttonSmsToLawsuit = itemView.findViewById(R.id.button_smsToLawsuit);
+      buttonCreateSmsLawsuit = itemView.findViewById(R.id.button_createLawsuit);
     }
 
     public void bind(SmsMessage sms) {
-      from.setText(sms.getSender());
+      from.setText(
+          String.format(itemView.getContext().getString(R.string.list_item_from), sms.getSender()));
       content.setText(sms.getBody());
-      receivedAt.setText(sms.getReceivedAt());
+      Calendar calendar = Calendar.getInstance();
+      Date receivedDate;
+
+      // Localize Hebrew date format
+      Locale local = new Locale("he");
+      try {
+        receivedDate = new SimpleDateFormat("dd/M/yyyy", local).parse(sms.getReceivedAt());
+        calendar.setTime(receivedDate);
+        receivedAt.setText(
+            String.format(
+                itemView.getContext().getString(R.string.list_item_date),
+                calendar.get(Calendar.DAY_OF_MONTH),
+                calendar.getDisplayName(Calendar.MONTH, Calendar.LONG, local),
+                calendar.get(Calendar.YEAR)));
+      } catch (ParseException e) {
+        Log.e(TAG, "Error parsing sms.ReceivedAt() to Date object\n" + e.getMessage());
+      }
+
       // Click on a spam message, from there (with message's details) move to the lawsuitPdfActivity
       // TODO: See which more fields in the lawsuit form can be understood from the SMS / other
       // DATA.
-      buttonSmsToLawsuit.setOnClickListener(
+      buttonCreateSmsLawsuit.setOnClickListener(
           view -> {
-            FirestoreClient client = new FirestoreClient();
-            client.writeMessage(sms, FirestoreClient.USER_SUGGEST_COLLECTION);
-            Toast.makeText(view.getContext(), "suggested", Toast.LENGTH_SHORT).show();
+            // Suggest Spam
```

Comment:
Can you describe in which cases this should be uncommented?

Reply:
Actually I think it'll be preferable to remove these lines out for now.
I'll post in the channel, it's one of the subjects we need to discuss about.